apps/server/src/shared/repo/schoolexternaltool/school-external-tool.repo.ts

```diff
@@ -43,6 +44,24 @@ export class SchoolExternalToolRepo extends BaseDORepo<
 		return count;
 	}
 
+	async find(query: Partial<SchoolExternalToolDO>): Promise<SchoolExternalToolDO[]> {
```

Comment:
The `Partial<T>` helper makes all properties optional. Is this intended?

Reply:
I created another type for the service and repository to show which properties can be used to find the object